src/main/java/frc/robot/subsystems/color/patterns/BIFlag.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.color.patterns;
+
+import frc.robot.subsystems.color.RGB;
+
+public class BIFlag {
```

Comment:
May as well make the pattern classes static, right?

Reply:
You can't sadly.